decentraland/App.js

```diff
@@ -46,8 +47,13 @@ Is the 'config/default.js' file correct?`;
   }
 
   async _loadAccount() {
-    const account = await recoverAccount();
-    if (account != null) store.dispatch(setAccount(account));
+    const account = await retrieveEphemeralAccount();
+    store.dispatch(setAccount(account));
```

Comment:
Noting that I need to check how we handle `null` after this change

Reply:
Any thoughts? I've removed the `if` because the default state for the account is `null`.

Reply:
That makes sense to me. We can go with the way you have it now. Resolving this.